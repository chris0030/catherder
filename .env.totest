A=1
B=2
C=3
E=7
F=5
ARGH=6