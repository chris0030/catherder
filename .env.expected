A=1
B=2
C=4
E=6
F=7
TEST=bloop